### PR TITLE
Appnexus: Remove audio type and Improve test coverage

### PR DESCRIPTION
**Pull request by Sonali-More-Xandr**

### CODE REVIEW

File: adapters/appnexus/appnexustest/exemplary/simple-banner.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
   "mockBidRequest": {
     "id": "test-request-id",
+    "app": {
```
Comment (mansinahar) [inline adapters/appnexus/appnexustest/exemplary/simple-banner.json:4]: Shouldn't the `source` and `version` values be inside `app.ext.prebid`? `app` itself doesn't support a `prebid` field

Comment (Sonali-More-Xandr) [reply adapters/appnexus/appnexustest/exemplary/simple-banner.json:4]: Updated the request in [ed9d44d](https://github.com/prebid/prebid-server/pull/2726/commits/ed9d44d7c86eb9356c76a8f2a9eceb4969e825ec)

**Review by onkarvhanumante — APPROVED** [review]

File: adapters/appnexus/appnexus_test.go [hunk 2]
```diff
@@ -30,6 +31,13 @@ func TestMemberQueryParam(t *testing.T) {
 	}
 }
 
+func TestBuilder(t *testing.T) {
```
Comment (mansinahar) [inline adapters/appnexus/appnexus_test.go:34]: Please consider updating the name of this function to something like `TestBuilderWithPlatformID` as we're specifically trying to test a builder where we're passing a custom platform ID.

Also please add assertion for `adapter.hbsource` to ensure platform ID is correctly set. You can do so as follows:
```
assert.NotNil(t, bidder)
assert.Equal(t, 3, (*bidder.(*adapter)).hbSource)
```

Comment (Sonali-More-Xandr) [reply adapters/appnexus/appnexus_test.go]: Updated assertions in [9a7e773](https://github.com/prebid/prebid-server/pull/2726/commits/9a7e7738aa835c0af2cf07f4195d827fe635fe82)

File: adapters/appnexus/appnexustest/supplemental/invalid-bid-type.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "app": {
+      "prebid": {
```
Comment (mansinahar) [inline adapters/appnexus/appnexustest/supplemental/invalid-bid-type.json:5]: Ughh! I missed this before but you need to move this to `app.ext.prebid` as well like you did in the other test.

Comment (Sonali-More-Xandr) [reply adapters/appnexus/appnexustest/supplemental/invalid-bid-type.json]: oh, I missed it. Updated it in [d9903b5](https://github.com/prebid/prebid-server/pull/2726/commits/d9903b50a4968f891769a5cecc3924366b660cb4)

**Review by mansinahar — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**